iOS/src/toga_iOS/widgets/canvas.py

```diff
@@ -15,7 +15,7 @@
 )
 from travertino.size import at_least
 
-from toga.colors import BLACK, TRANSPARENT, color
+from toga.colors import BLACK, TRANSPARENT, color as named_color
```

Comment:
Worth noting that this is a little misleading - `color` here isn't *just* named colors - it's a toga color that will accept any valid string.